augur/cli/run.py

```diff
@@ -32,13 +36,29 @@ def cli(disable_housekeeper, skip_cleanup):
     else:
         logger.debug("Skipping process cleanup")
 
+    if logstash or logstash_with_cleanup:
+        augur_home = os.getenv('AUGUR_HOME', "")
+        if logstash_with_cleanup:
+            print("Cleaning old workers errors...")
+            with open(augur_home + "/log_analysis/http/empty_index.html") as f:
+                lines = f.readlines()
+            with open(augur_home + "/log_analysis/http/index.html", "w") as f1:
+                f1.writelines(lines)
+            print("All previous workers errors got deleted.")
+
+        elasticsearch_path = os.getenv('ELASTIC_SEARCH_PATH', "")
```

Comment:
Are there any sensible defaults for this environment variable? Like running on localhost, for example? (Same for the `LOGSTASH_PATH`)

Reply:
@ccarterlandis this is the path to the local elasticsearch/logstash bin. I left it as a variable so that one will not need to stick to specific convention. If you prefer I can document how to install and set a default.

Reply:
I think documenting a default would be fine!!